### DISCUSSION TITLE
confirmation when deleting g-code files

### DISCUSSION THREAD

**Discussion by Be4Prints** [mainsail-crew/mainsail#1831] [General]
Is there a way to disable confirmation when deleting g-code files? Since I still have my files on my PC anyway, it doesn't matter if I accidentally delete them on the printer

**Comment by meteyou** [comment]
No, there is no option to turn it off, and I don't think we will add one for this function because we already have too many options to enable/disable confirmations...

**Reply by Be4Prints** [reply]
Okay, thank you for the answer. Even if the reasoning seems a bit strange to me.
It doesn't matter, if there are too many options with buttons, because you only set it up once. But you always have to confirm, when ever you have to delete a file.
Thank you anyway

**Reply by meteyou** [reply]
you can also select multiple files and delete many files at once... i don't see any reason to "disable" this confirmation...
